Mathlib/Data/Complex/Module.lean

```diff
@@ -337,6 +337,10 @@ section lift
 
 variable {A : Type _} [Ring A] [Algebra ℝ A]
 
+-- Porting note: added for `liftAux` proof
+local instance {R A} [CommSemiring R] [Semiring A] [Algebra R A] : SMulCommClass R A A := by
```

Comment:
I think we should backport this and add it globally. It's a prop-only instance and the most common use of smul_comm_clasa.

Reply:
Like this? https://github.com/leanprover-community/mathlib/pull/18907